src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+import javafx.application.Application;
+
+/**
+ * A launcher class to workaround classpath issues.
+ */
+public class Launcher {
+    public static void main(String[] args) {
+        Application.launch(Duke.class, args);
+    }
+}
```

Comment:
Not explicitly in the coding standard, but as highlighted by GitHub here and based on personal experience, I think it is tradition/standard in many places to end a file with a new (blank) line.